graphene/contrib/django/filter/fields.py

```diff
@@ -26,7 +27,13 @@ def get_queryset(self, qs, args, info):
         filter_kwargs = self.get_filter_kwargs(args)
         order = self.get_order(args)
         if order:
-            qs = qs.order_by(order)
+            qs = qs.order_by(to_snake_case(order))
```

Comment:
@slorg1 you're probably right about moving the snake case translation into `get_order` as that is probably cleaner. I also don't think graphene currently supports ordering by multiple fields. Possibly raise that as a separate issue/PR with a failing test case?